crd/multitenancy/manifests/multitenancy.acn.azure.com_podnetworkinstances.yaml

```diff
@@ -71,6 +71,12 @@ spec:
                 items:
                   type: string
                 type: array
+              status:
+                description: PNIStatus indicates the status of PNI
+                enum:
+                - Ready
+                - Error
```

Comment:
as discussed offline - the "Error" status is too opaque and we should be more descriptive here. 
the intent is to bubble up if the PNI has been attempted (vs still pending), and if it failed to provide context to the observer of this CRD on whether they can/need to take some action.